consensus/src/chained_bft/event_processor.rs

```diff
@@ -657,6 +669,14 @@ impl<T: Payload> EventProcessor<T> {
             self.author,
             ledger_info_placeholder,
             self.block_store.signer(),
+            // Note that SyncInfo carried by VoteMsg cannot include PacemakerTimeoutCertificate,
+            // because PacemakerTimeoutCertificate might include the VoteMsg, which would create
+            // gigantic chains of SyncInfo.
```

Comment:
This is temporary right? The TimeoutCertificate only carries the round signature

Reply:
Yes, exactly.